src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IArgument.cs

```diff
@@ -2212,12 +2212,15 @@ static P M1()
                 // (4,12): error CS0518: Predefined type 'System.Object' is not defined or imported
                 //     static P M1()
                 Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "P").WithArguments("System.Object").WithLocation(4, 12),
+                // (9,7): error CS0518: Predefined type 'System.Boolean' is not defined or imported
+                //     P(bool? x = true)
+                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "bool").WithArguments("System.Boolean").WithLocation(9, 7),
                 // (9,7): error CS0518: Predefined type 'System.Nullable`1' is not defined or imported
                 //     P(bool? x = true)
                 Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "bool?").WithArguments("System.Nullable`1").WithLocation(9, 7),
-                // (9,7): error CS0518: Predefined type 'System.Boolean' is not defined or imported
+                // (9,7): error CS0518: Predefined type 'System.Nullable`1' is not defined or imported
```

Comment:
>// (9,7): error CS0518: Predefined type 'System.Nullable`1' is not defined or imported [](start = 16, length = 86)

This looks like a new duplicate error.

Reply:
Added comment to investigate.

---
In reply to: [171140841](https://github.com/dotnet/roslyn/pull/25087#discussion_r171140841) [](ancestors = 171140841)